core/domain/html_validation_service_test.py

```diff
@@ -973,3 +975,72 @@ def test_validate_customization_args_in_tag(self):
+
+    def test_add_dimensions_to_image_tags(self):
+        test_cases = [{
+            'html_content': (
+                '<p><oppia-noninteractive-image filepath-with-value="&amp;quot;'
+                'abc1.png&amp;quot;"></oppia-noninteractive-image>Hello this'
+                ' is test case to check that dimensions are added to the oppia'
+                ' noninteractive image tags.</p>'
+            ),
+            'expected_output': (
+                u'<p><oppia-noninteractive-image filepath-with-value='
+                '"&amp;quot;abc1_height_32_width_32.png&amp;'
+                'quot;"></oppia-noninteractive-image>Hello this is test case'
+                ' to check that dimensions are added to the oppia '
+                'noninteractive image tags.</p>'
+            )
+        }, {
+            'html_content': (
+                '<p><oppia-noninteractive-image filepath-with-value="&amp;quot;'
+                'abc2.png&amp;quot;"></oppia-noninteractive-image>Hello this'
+                ' is test case to check that dimensions are added to the oppia'
+                ' noninteractive image tags.<oppia-noninteractive-image '
+                'filepath-with-value="&amp;quot;abc3.png&amp;quot;">'
+                '</oppia-noninteractive-image></p>'
+            ),
+            'expected_output': (
+                u'<p><oppia-noninteractive-image filepath-with-value="'
+                '&amp;quot;abc2_height_32_width_32.png&amp;quot;">'
+                '</oppia-noninteractive-image>Hello this is test case '
+                'to check that dimensions are added to the oppia'
+                ' noninteractive image tags.<oppia-noninteractive-image '
+                'filepath-with-value="&amp;quot;abc3_height_32_width_32.png'
+                '&amp;quot;"></oppia-noninteractive-image></p>'
+            )
+        }, {
+            'html_content': (
+                '<p>Hey this is a test case with no images.</p>'
+            ),
+            'expected_output': (
+                u'<p>Hey this is a test case with no images.</p>'
+            )
+        }]
+
+        EXP_ID = 'eid'
+        OWNER_ID = 'Admin'
+
+        with open(os.path.join(feconf.TESTS_DATA_DIR, 'img.png')) as f:
+            raw_image = f.read()
+        fs = fs_domain.AbstractFileSystem(
+            fs_domain.ExplorationFileSystem(EXP_ID))
+        fs.commit(OWNER_ID, 'abc1.png', raw_image, mimetype='image/png')
+        fs.commit(OWNER_ID, 'abc2.png', raw_image, mimetype='image/png')
+        fs.commit(OWNER_ID, 'abc3.png', raw_image, mimetype='image/png')
+
+        for test_case in test_cases:
+            self.assertEqual(
+                html_validation_service.add_dimensions_to_image_tags( # pylint: disable=line-too-long
+                    EXP_ID, test_case['html_content']),
+                test_case['expected_output'])
+
+    def test_regenerate_image_filename_using_dimensions(self):
+        FILENAME = 'abc.png'
```

Comment:
I've asked at least twice for these attributes to be inlined. By that I mean:

```
        regenerated_name = (
            html_validation_service.regenerate_image_filename_using_dimensions(
                'abc.png', 45, 45))
```

Is there some reason not to do that that I'm missing?

Reply:
I am sorry, i misinterpreted 'inline'. My bad. Will update it now.